code/modules/paperwork/faxmachine.dm

```diff
@@ -283,34 +283,25 @@ var/global/list/fax_blacklist = list()
 
 	use_power(200)
 
-	var/obj/item/rcvdcopy
-	if(istype(copyitem, /obj/item/paper))
-		rcvdcopy = copy(copyitem)
-	else if(istype(copyitem, /obj/item/photo))
-		rcvdcopy = photocopy(copyitem)
-	else if(istype(copyitem, /obj/item/paper_bundle))
-		rcvdcopy = bundlecopy(copyitem)
-	else
+	if((!istype(copyitem, /obj/item/paper))&&(!istype(copyitem, /obj/item/photo))&&(!istype(copyitem, /obj/item/paper_bundle)))
```

Comment:
Good heavens almighty, man. Styling.

Reply:
I'm not sure what you mean by styling. Just the mere fact it's all compressed or...?

Reply:
Oh, nah I think it's that there ain't no spaces around `&&` as well as the unnecessary parentheses around the `!istype()` checks.

Could maybe be written as
`if(!(istype(copyitem, /obj/item/paper) || istype(copyitem, /obj/item/paper_bundle) || istype(copyitem, /obj/item/photo))`

Reply:
Just did a change before you posted that. I could just write it that way.

Reply:
My bad, I could just about sense you typing away and tried to post it asap :rofl: